composer.json

```diff
@@ -7,7 +7,7 @@
         "php": ">=5.4.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4|^5",
+        "phpunit/phpunit": "^4.8|^5.7",
```

Comment:
I'm not sure which one we actually need, for testing STDERR output, we need at least `5.1`. `4.0` does at not cover everything we need as Amp tests with `--prefer-lowest` have shown, so I just used the latest one here.